command/execute.go

```diff
@@ -75,13 +75,13 @@ var Datasources = map[string]packersdk.Datasource{
 
 var pluginRegexp = regexp.MustCompile("packer-(builder|post-processor|provisioner|datasource)-(.+)")
 
-func (c *PluginCommand) Run(args []string) int {
+func (c *ExecuteCommand) Run(args []string) int {
 	// This is an internal call (users should not call this directly) so we're
 	// not going to do much input validation. If there's a problem we'll often
 	// just crash. Error handling should be added to facilitate debugging.
 	log.Printf("args: %#v", args)
 	if len(args) != 1 {
-		c.Ui.Error("Wrong number of args")
+		c.Ui.Error("Wrong number of args - this command is not meant to be manually invoked.")
```

Comment:
Instead of displaying `Wrong number of args` what about displaying c.Help() which already includes messaging that the command should not be called directly.

```suggestion
		c.Ui.Error(c.Help())
```

The output would look like 
```shell
~>  ./bin/packer execute foo

Usage: packer execute PLUGIN

  Runs an internally-compiled version of a plugin from the packer binary.

  NOTE: this is an internal command and you should not call it yourself.
```

Reply:
Oh yeah fair, good call